packages/editor/src/actions/panes.actions.ts

```diff
@@ -84,5 +84,18 @@ export const panesActions = {
             type: panesActions.MOVE_PANE,
             data: { fromIndex, toIndex }
         };
+    },
+    OUTBOUND_LINK_CLICK: 'OUTBOUND_LINK_CLICK',
+    outboundLinkClick(url: string) {
+        return {
+            type: panesActions.OUTBOUND_LINK_CLICK,
+            data: url
+        };
+    },
+    OUTBOUND_LINK_CLICK_SUCCESS: 'OUTBOUND_LINK_CLICK_SUCCESS',
```

Comment:
Do we need this `OUTBOUND_LINK_CLICK_SUCCESS`?

Reply:
It is being called on successful click which takes you out of the studio. It is a good indicator that everything went well, so we can keep it.